app/src/main/java/com/android/sample/ui/swipePage/SwipePage.kt

```diff
@@ -392,26 +390,6 @@ private fun ImageDescription(name: String, tag: String) {
   }
 }
 
-/**
- * Composable for the Tags of the recipe
- *
- * @param tag - Tag Name
- */
-@Composable
-private fun Tag(tag: String) {
-  Box(
-      modifier =
-          Modifier.background(
-                  color = tagBackground,
-                  shape = RoundedCornerShape(16.dp)) // Smooth rounded corners
-              .padding(horizontal = 12.dp, vertical = 4.dp) // Padding for inside spacing
-      ) {
-        Text(
-            text = tag, fontSize = 14.sp, color = Color.White // Text color
-            )
-      }
-}
-
```

Comment:
Nice so this is the removed function that you have seen that you could use elsewhere than in the `SwipePage` function, nice catch !!